### ISSUE TITLE
php tests failing

### ISSUE THREAD

**Issue by reiddraper** [basho/riak_cs#540]
I'm trying to review #539, and am having trouble running the PHP integration tests. Specifically, I'm getting this error:

``` bash
➜  php git:(a1766b7) ./phpunit

Parse error: parse error in /Users/reid/Documents/repos/basho/riak_cs/client_tests/php/tests/S3ClientTest.php on line 41

➜  php git:(a1766b7) php --version
PHP 5.3.15 with Suhosin-Patch (cli) (built: Aug 24 2012 17:45:44)
Copyright (c) 1997-2012 The PHP Group
Zend Engine v2.3.0, Copyright (c) 1998-2012 Zend Technologies
```

I _think_ I have everything set up properly, but could be missing something.

**Comment by reiddraper** [comment]
cc @ksauzz

**Comment by hectcastro** [comment]
I installed PHP54 to get this running:

``` bash
$ php --version
PHP 5.4.13 (cli) (built: Apr 19 2013 10:58:54)
Copyright (c) 1997-2013 The PHP Group
Zend Engine v2.4.0, Copyright (c) 1998-2013 Zend Technologies
$ composer --version
Composer version 1.0.0-alpha6
```

**Comment by reiddraper** [comment]
Using 5.4.14 now:

```
PHPUnit 3.7.19 by Sebastian Bergmann.

Configuration read from /Users/reid/Documents/repos/basho/riak_cs/client_tests/php/phpunit.xml.dist

..[31;1mE[0m[31;1mE[0m.

Time: 0 seconds, Memory: 7.75Mb

There were 2 errors:

1) S3ClientTest::testPutGetDeleteObject
Invalid argument supplied for foreach()

/Users/reid/Documents/repos/basho/riak_cs/client_tests/php/tests/S3ClientTest.php:42

2) S3ClientTest::testBucketACL
Invalid argument supplied for foreach()

/Users/reid/Documents/repos/basho/riak_cs/client_tests/php/tests/S3ClientTest.php:42

[37;41m[2KFAILURES!
[0m[37;41m[2KTests: 5, Assertions: 10, Errors: 2.
[0m[2K
```

**Comment by ksauzz** [comment]
I reproduced the issue. php-test can't run with aws-sdk-php-2.2 or later. I'll try to update php-tests.

**Comment by reiddraper** [comment]
Another php README note, I had to use `php -d detect_unicode=Off` when installing composer. OSX 10.7. Though that was when I was using php 5.3.x.

**Comment by hectcastro** [comment]
Added this note to https://github.com/basho/riak_cs/issues/540.

**Comment by reiddraper** [comment]
s/#540/#539

**Comment by reiddraper** [comment]
Fix in #541.
